tools/StaticAnalysis/Exceptions/Az.Monitor/SignatureIssues.csv

```diff
@@ -6,4 +6,5 @@
 "Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll","Microsoft.Azure.Commands.Insights.ScheduledQueryRules.NewScheduledQueryRuleLogMetricTriggerCommand","New-AzScheduledQueryRuleLogMetricTrigger","1","8100","New-AzScheduledQueryRuleLogMetricTrigger Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll","Microsoft.Azure.Commands.Insights.ScheduledQueryRules.NewScheduledQueryRuleScheduleCommand","New-AzScheduledQueryRuleSchedule","1","8100","New-AzScheduledQueryRuleSchedule Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll","Microsoft.Azure.Commands.Insights.ScheduledQueryRules.NewScheduledQueryRuleSourceCommand","New-AzScheduledQueryRuleSource","1","8100","New-AzScheduledQueryRuleSource Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
-"Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll","Microsoft.Azure.Commands.Insights.ScheduledQueryRules.NewScheduledQueryRuleTriggerConditionCommand","New-AzScheduledQueryRuleTriggerCondition","1","8100","New-AzScheduledQueryRuleTriggerCondition Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
\ No newline at end of file
+"Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll","Microsoft.Azure.Commands.Insights.ScheduledQueryRules.NewScheduledQueryRuleTriggerConditionCommand","New-AzScheduledQueryRuleTriggerCondition","1","8100","New-AzScheduledQueryRuleTriggerCondition Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll","Microsoft.Azure.Commands.Insights.Metrics.GetAzureRmMetricCommand","Get-AzMetric","0","3030","The generic type for 'property Count' has been changed from 'System.Nullable1[System.Int64]' to 'System.Nullable1[System.Double]'. ","Change the generic type for 'property Count' back to 'System.Nullable`1[System.Int64]'."
```

Comment:
@gucalder you added this line to the `SignatureIssues.csv` file when it should have gone in the `BreakingChangeIssues.csv` file -- this would cause the build to fail since the files are parsed differently due to the differences in their columns